servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,7 +1,7 @@
 [[removing_service-mesh]]
 = Removing {ProductName}
 
-The following command removes the {ProductShortName} from an existing installation.  You can run these commands from any host with access to the cluster.  Ensure you are logged in as a cluster admin before running the following commands.
+The following command removes the {ProductShortName} from an existing installation. You can run these commands from any host with access to the cluster. Ensure you are logged in as a cluster admin before running the following commands.
```

Comment:
The following command removes...

...before running the following commands.

Seems repetitive.

The following command removes the {ProductShortName} from an existing installation. Provided you are logged in as a cluster admin, you can run these commands from any host with access to the cluster. ?